src/Components/TwoOneOneNCComponents/TwoOneOneHeader/TwoOneOneHeaderNC.tsx

```diff
@@ -0,0 +1,202 @@
+import { useContext, useState, useEffect } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { AppBar, MenuItem, Select, Link, IconButton, Dialog, SelectChangeEvent } from '@mui/material';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { useConfig } from '../../Config/configHook';
+import { Context } from '../../Wrapper/Wrapper';
+import NCtwoOneOneMFBLogo from '../../../Assets/States/NC/WhiteLabels/TwoOneOneAssets/nc211-MFB-CTD-2025.png';
+import twoOneOneNCLinks from '../../../Assets/States/NC/WhiteLabels/TwoOneOneAssets/twoOneOneNCLink';
+import LanguageIcon from '@mui/icons-material/Language';
+import ShareIcon from '@mui/icons-material/Share';
+import MenuIcon from '@mui/icons-material/Menu';
+import CloseIcon from '@mui/icons-material/Close';
+
+import './TwoOneOneHeaderNC.css';
+import TwoOneOneShareNC from '../TwoOneOneShare/TwoOneOneShareNC';
+
+type LanguageOptions = {
+  [key: string]: string;
+};
+
+const TwoOneOneHeaderNC = () => {
+  //this is so that when the users click on the cobranded logo, they're navigated back to step-1
+  const { formData, locale, selectLanguage } = useContext(Context);
+  const languageOptions = useConfig<LanguageOptions>('language_options');
+  const queryString = formData.immutableReferrer ? `?referrer=${formData.immutableReferrer}` : '';
```

Comment:
We have a better way to do this now. Could you please update CO as well.
```suggestion
  const queryString = useQueryString();
```

Reply:
done